pandas/io/packers.py

```diff
@@ -255,17 +264,38 @@ def unconvert(values, dtype, compress=None):
     if not as_is_ext:
         values = values.encode('latin1')
 
-    if compress == u'zlib':
-        import zlib
-        values = zlib.decompress(values)
-        return np.frombuffer(values, dtype=dtype)
-
-    elif compress == u'blosc':
-        import blosc
-        values = blosc.decompress(values)
-        return np.frombuffer(values, dtype=dtype)
+    if compress:
+        if compress == u'zlib':
```

Comment:
I would try to import these at the top of the file and just set variables (``_ZLIB_INSTALLED``, ``_BLOSC_INSTALLED``); and you can do whatever imports you need there.